static/app/components/events/interfaces/spans/inlineDocs.tsx

```diff
@@ -123,9 +39,183 @@ class InlineDocs extends Component<Props, State> {
+          {code: <code />}
+        )}
+      </p>
+      <StyledCodeSnippet language="python" dark>
+        {`import sentry_sdk
+
+def eat_slice(slice):
+    ...
+
+def eat_pizza(pizza):
+    with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
+        while pizza.slices > 0:
+            eat_slice(pizza.slices.pop())
+`}
+      </StyledCodeSnippet>
+      <p>
+        <strong>{t('Adding More Spans to the Transaction')}</strong>
+      </p>
+      <p>
+        {t(
+          'If you want to have more fine-grained performance monitoring, you can add child spans to your transaction, which can be done by either:'
+        )}
+      </p>
+      <ul>
+        <li>{t('Using a context manager or')}</li>
+        <li>{t('Using a decorator, (this works on sync and async functions)')}</li>
+      </ul>
+      <p>
+        {tct(
+          'Calling a [code:sentry_sdk.start_span()] will find the current active transaction and attach the span to it.',
+          {code: <code />}
+        )}
+      </p>
+      <StyledCodeSnippet language="python" dark>
+        {`import sentry_sdk
+
+def eat_slice(slice):
+...
+
+def eat_pizza(pizza):
+    with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
+        while pizza.slices > 0:
+            with sentry_sdk.start_span(description="Eat Slice"):
+                eat_slice(pizza.slices.pop())
+`}
+      </StyledCodeSnippet>
+      <StyledCodeSnippet language="python" dark>
+        {`import sentry_sdk
+
+@sentry_sdk.trace
+def eat_slice(slice):
+    ...
+
+def eat_pizza(pizza):
+    with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
+        while pizza.slices > 0:
+            eat_slice(pizza.slices.pop())
+`}
+      </StyledCodeSnippet>
+      <p>
+        {tct(
+          `For in-depth instructions on setting up tracing, view [docLink:our documentation].`,
+          {
+            docLink: <a href="https://docs.sentry.io/platforms/python/tracing/" />,
+          }
+        )}
+      </p>
+    </DocumentationWrapper>
+  );
+}
+
+function NodeDocs() {
+  return (
+    <DocumentationWrapper>
+      <p>
+        {t(
+          'To manually instrument a specific region of your code, you can create a transaction to capture it.'
+        )}
+      </p>
+      <p>
+        {tct(
+          'The following example creates a transaction for a part of the code that contains an expensive operation (for example, [code:processItem]), and sends the result to Sentry:',
+          {code: <code />}
+        )}
+      </p>
+      <StyledCodeSnippet language="javascript" dark>
+        {`app.use(function processItems(req, res, next) {
+  const item = getFromQueue();
+  const transaction = Sentry.startTransaction({
+    op: "task",
+    name: item.getTransaction(),
+  });
+
+  // processItem may create more spans internally (see next examples)
+  processItem(item, transaction).then(() => {
+    transaction.finish();
+    next();
+  });
+});
+`}
```

Comment:
why the brackets and just not a plain text?

Reply:
This gives me better control over indentation and I do not need to care about escaping specific characters like parenthesis or brackets.